configure.ac

```diff
@@ -0,0 +1,191 @@
+AC_PREREQ([2.69])
+AC_INIT([rkt], [0.1], [https://github.com/coreos/rkt/issues])
+
+# Sets parameters
+
+# RKT source distribution path
+AC_ARG_WITH(org-path,
+        AS_HELP_STRING([--with-org-path=PATH],
+                [Path to rkt source distribution]),
```

Comment:
Please add an information about default value in help string, like for example: `[Path to rkt source distribution (default: 'github.com/coreos')]` or `[--with-org-path=github.com/coreos]`

Anyway, I don't think that --with-org-path option is that really useful. And it wasn't overridable in build script.